config/salary_frequency_conversions.yml

```diff
@@ -0,0 +1,10 @@
+salary_frequencies:
```

Comment:
This config is replicated elsewhere. We will need to think about refactoring this _config_ to have a single source of truth for _salary_frequency_. No action needed here. Just a reminder of a potential need for later refactoring.